### ISSUE TITLE
Redesign Lightning wallet connect UI

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#5604]
_The goal of this issue is to bring in discussion around potential UI improvements, suggested improvements have to be discussed, ironed out, so this issue shouldn't be worked on yet._

Context:
Our lightning wallet UI is quite different than our create a wallet UI. The pill UI doesn't really allow us to showcase a great amount of options that are possible, and it seems that with plugins a whole set of connect external lightnign wallets is coming into the place (Breez, Alby, Blink, LDK).

I think as our selection of options to use Lightning (in a custodial, or self-custodial way) grow, our UI has to cater for those options and guide users through setting up an adequate one.

Currently:
Our UI for lightning looks like this

<img width="883" alt="Screenshot 2023-12-22 at 21 02 10" src="https://github.com/btcpayserver/btcpayserver/assets/36959754/a6dbb50d-ce6a-47dc-b2e8-8bf1f656c2ee">

Our UI for creating an on-chain wallet is way more accessible and guides users to tailored options and will help us unify the UI components a bit more.

<img width="904" alt="Screenshot 2023-12-22 at 21 03 31" src="https://github.com/btcpayserver/btcpayserver/assets/36959754/245505c4-8072-44fb-80e4-7f41df21c937">

It can be: (obviously we need to discuss these), 
- Start a lightning node
- Deploy LN node
- Deploy LDK
 -Deploy Breez
- Connect a lightning wallet
   -  Connect via plugin (where we showcase all lightning plugins under payment method category, even if pliugins are not published)
-  Connect external node (advanced)

**Comment by dstrukt** [comment]
Makes sense to tackle this, it has been a while since we've touched this page + a lot of the wallet views. Will make a note to discuss in the next design call!

**Comment by DoktorShift** [comment]
I Just searched for an open issue to connect lnbits as a funding source for the lightning payments inside btcpay.

Nice to See you are working on solutions to accept more Implementation options for the ln payments then the actually Provided Options (custom node and internal node)

Should i open an issue which points directly to the usecase to connect a lnbits Wallet as funding source?

Cheers

**Comment by dstrukt** [comment]
@DoktorShift Sure, either works - if it's easier, you can drop in this Issue as well!

**Comment by pavlenex** [comment]
Related https://github.com/btcpayserver/btcpayserver/discussions/6268
